docs/usage/updates-strategy.md

```diff
@@ -101,3 +101,35 @@ Reboot windows can be separately configured in multiple snippets, as long as eac
  * `length_minutes`: non-zero window duration, in minutes
 
 For convenience, multiple entries can be defined with overlapping times, and each window definition is allowed to cross day and week boundaries (wrapping to the next day).
+
```

Comment:
Let's add a `## Timezone configuration` header so that's it's easier to link to.